src/register/pmpcfgx.rs

```diff
@@ -66,30 +66,10 @@ impl PmpCfg {
 
 pub struct PmpCfgCsr {
     #[cfg(riscv32)]
-    cfgs: [PmpCfg; 4],
+    pub cfgs: [PmpCfg; 4],
 
     #[cfg(riscv64)]
-    cfgs: [PmpCfg; 8],
-}
-
-impl PmpCfgCsr {
-    pub fn get_cfg(&self, index: usize) -> PmpCfg {
```

Comment:
Were there no users of this getter function?

Reply:
Not that I can find.

Reply:
:+1: